packages/core/src/_common/base/socket_base.js

```diff
@@ -310,6 +310,18 @@ const BinarySocketBase = (() => {
 
     const tradingServers = () => deriv_api.send({ platform: 'mt5', trading_servers: 1 });
 
+    const triggerMt5AccountBO = ({ email }) =>
```

Comment:
nitpick: Can we call it `triggerMt5DryRun` instead?

Reply:
Done